pkg/cmd/inspect/resource.go

```diff
@@ -27,6 +27,9 @@ import (
 const (
 	clusterScopedResourcesDirname = "cluster-scoped-resources"
 	namespaceResourcesDirname     = "namespaces"
+
+	configResourceDataKey   = "/cluster-scoped-resources/config.openshift.io"
+	operatorResourceDataKey = "/cluster-scoped-resources/operator.openshift.io"
```

Comment:
Should we not string build here? 
This will keep clusterScopedResourcesDirname from getting out of sync with the DataKey's

Reply:
I don't think we mind if they get out of sync. Primary purpose of the key is to create a unique identifier to keep track of what we've collected in the resourceContext